static/js/components/ManageDataForm.jsx

```diff
@@ -125,47 +127,39 @@ const useStyles = makeStyles(() => ({
   groupSelect: {
     width: "20rem",
   },
-  comments: {
-    justify: "flex-end",
-  },
-  container: {
-    display: "flex",
-    flexDirection: "row",
-    justifyContent: "flex-start",
-  },
 }));
 
 const SpectrumRow = ({ rowData, route }) => {
   const styles = useSourceStyles();
-  const classes = useStyles();
-
+  const colSpan = rowData.length + 1;
   return (
-    <div className={classes.container}>
-      <Grid container direction="row" justify="center" alignItems="center">
-        <Grid item className={styles.photometryContainer}>
-          <Suspense fallback={<div>Loading spectroscopy plot...</div>}>
-            <Plot
-              className={styles.plot}
-              // eslint-disable-next-line react/prop-types
-              url={`/api/internal/plot/spectroscopy/${route.id}?spectrumID=${rowData[0]}`}
+    <TableRow>
+      <TableCell colSpan={colSpan}>
```

Comment:
The use of the `TableRow` and `TableCell` components here along with the `colSpan` parameter is what gets the pullout content to actually span the whole width of the table. I think the fact that this is part of table complicated how the CSS was being interpreted and why it wouldn't just fill it with a standard `width: 100%` rule. 

Also, you'll notice I got rid of the div with the flexbox styling. The `Grid` component is actually just an abstraction over a flexbox container so I think having another div over the whole thing that is also a flexbox is unnecessary and doesn't really affect anything.

Reply:
Yeah, that makes a lot of sense. Also, I'm usually too scared to remove divs because then all hell breaks loose. But I will get this whole JavaScript thing eventually... 

Thanks for the assistance!